src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleTypeSymbol.vb

```diff
@@ -765,9 +765,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                                                                                container.Name & "." & TupleMemberName(fieldRemainder)))
 
                     Dim defaultName As String = TupleMemberName(i + 1)
-                    ' Add a field with default name if the given name Is different
+                    ' Add a field with default name if the given name is different
                     If Name <> defaultName Then
-                        members.Add(New TupleErrorFieldSymbol(Me, defaultName, -members.Count - 1, Nothing, _elementTypes(i), diagnosticInfo))
+                        members.Add(New TupleErrorFieldSymbol(Me, defaultName, i, Nothing, _elementTypes(i), diagnosticInfo))
```

Comment:
>i [](start = 79, length = 1)

This change doesn't look consistent with the rest of the implementation and with comments.